Makefile

```diff
@@ -13,8 +13,8 @@ all:
 # deps will clean out the vendor directory and use go mod for a fresh install
 deps:
 	rm -rf vendor
-	go mod vendor
-	go mod tidy
+	GOPROXY="https://proxy.golang.org" go mod vendor -v
+	go mod tidy -v
```

Comment:
Thanks @benjaminch! Just one more thing though, what do you think about combining these two commands into one like:

```
GOPROXY="https://proxy.golang.org" go mod vendor -v && go mod tidy -v
```

The reason being, that `go mod tidy` can also potentially install any missing dependencies and so it will be good to supply it with the `GOPROXY` variable as well.

Ref (https://github.com/golang/go/wiki/Modules):

go mod tidy — Prune any no-longer-needed dependencies from go.mod and add any dependencies needed for other combinations of OS, architecture, and build tags

Reply:
No worries, just updated :) Thanks !